tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -682,6 +821,24 @@ def test_llmobs_langchain_with_openai_disabled(self):
         self._call_openai_llm(OpenAI)
         self._assert_trace_structure_from_writer_call_args(["llm"])
 
+    @run_in_subprocess(env_overrides=openai_env_config)
+    def test_llmobs_langchain_with_embedding_model_openai_enabled(self):
+        from langchain_openai import OpenAIEmbeddings
+
+        patch(langchain=True, openai=True)
+        LLMObs.enable(ml_app="<ml-app-name>", integrations_enabled=False, agentless_enabled=True)
+        self._call_openai_embedding(OpenAIEmbeddings)
+        self._assert_trace_structure_from_writer_call_args(["workflow"])
```

Comment:
```suggestion
        self._assert_trace_structure_from_writer_call_args(["workflow", "embedding"])
```
Just noticed this, is this not being traced correctly?